src/blocks/skeleton/Skeleton.constants.ts

```diff
@@ -0,0 +1,3 @@
+import type { SkeletonProps } from './Skeleton.types';
+
+export const skeletonCSSPropsKeys: (keyof SkeletonProps)[] = ['borderRadius', 'height', 'margin', 'width'];
```

Comment:
I check out only height and width are passed down to html tag, you can also recheck and update this array by removing borderRadius and margin.